app/services/hyrax/versioning_service.rb

```diff
@@ -1,36 +1,35 @@
-require 'wings/models/file_node'
 require 'wings/services/file_node_builder'
 
 module Hyrax
   class VersioningService
     class << self
       # Make a version and record the version committer
-      # @param [ActiveFedora::File | Wings::FileNode] content
+      # @param [ActiveFedora::File | Hyrax::FileNode] content
```

Comment:
All changes in this file are related to the namespace change moving FileNode to Hyrax.